### DISCUSSION TITLE
Database

### DISCUSSION THREAD

**Discussion by meexz1** [evilsocket/opensnitch#708] [Q&A]
Where on linux does opensnitch saves its history/database if option in memory is selected? I can see number of hits even after restart.

**Comment by gustavo-iniguez-goya** [comment]
As far as I know it only stores data in memory: https://sqlite.org/inmemorydb.html

But under certain circusntances it could use a temporary file on disk. Maybe you can find it with: `$ ls /proc/$(pgrep opensnitch-ui)/fd/` while the GUI is opened.

> I can see number of hits even after restart.

Could you expand on that?
